### ISSUE TITLE
Update status should report a message for non-primary units.

### ISSUE THREAD

**Issue by MiaAltieri** [canonical/mongodb-operator#75]
## Problem:
TL;DR: The update status hook of the MongoDB charm incorrectly reports which unit is the "Replica Set Primary". **It reports the old primaries and the current primaries.**

You can see this on a real deployment
```
juju status 
ubuntu@mongo-ha:~/mongodb-operator$ juju status
Model  Controller  Cloud/Region         Version  SLA          Timestamp
neato  overlord    localhost/localhost  2.9.32   unsupported  13:15:07Z

App      Version  Status  Scale  Charm    Channel  Rev  Exposed  Message
mongodb           active      3  mongodb             0  no

Unit        Workload  Agent  Machine  Public address  Ports      Message
mongodb/0*  active    idle   0        10.30.220.92    27017/tcp    Replica set primary
mongodb/1   active    idle   1        10.30.220.211   27017/tcp
mongodb/2   active    idle   2        10.30.220.123   27017/tcp
```
and for some reason `mongodb/0` is no longer the primary and `mongodb/1` is the new primary, then `juju status` will show 
```
juju status 
ubuntu@mongo-ha:~/mongodb-operator$ juju status
Model  Controller  Cloud/Region         Version  SLA          Timestamp
neato  overlord    localhost/localhost  2.9.32   unsupported  13:15:07Z

App      Version  Status  Scale  Charm    Channel  Rev  Exposed  Message
mongodb           active      3  mongodb             0  no

Unit        Workload  Agent  Machine  Public address  Ports      Message
mongodb/0*  active    idle   0        10.30.220.92    27017/tcp    Replica set primary  <----- old status message
mongodb/1   active    idle   1        10.30.220.211   27017/tcp      Replica set primary
mongodb/2   active    idle   2        10.30.220.123   27017/tcp
```

This is **wrong** we would instead expect to see is:
```
juju status 
ubuntu@mongo-ha:~/mongodb-operator$ juju status
Model  Controller  Cloud/Region         Version  SLA          Timestamp
neato  overlord    localhost/localhost  2.9.32   unsupported  13:15:07Z

App      Version  Status  Scale  Charm    Channel  Rev  Exposed  Message
mongodb           active      3  mongodb             0  no

Unit        Workload  Agent  Machine  Public address  Ports      Message
mongodb/0*  active    idle   0        10.30.220.92    27017/tcp    
mongodb/1   active    idle   1        10.30.220.211   27017/tcp      Replica set primary <----- note only one unit is correctly reporting as primary. 
mongodb/2   active    idle   2        10.30.220.123   27017/tcp
```

## Solution:
Update the [update status-hook](https://github.com/canonical/mongodb-operator/blob/ec98e4a2dd2d4ea9ceb91afb2bd4fe2d3a5922cd/src/charm.py#L269-L279), to set an empty status "" for non-primary units. 

## Lingo:
`primary` - is the only member in the replica set that receives write operations
`leader` - is the juju unit that is marked as the leader
